cli.js

```diff
@@ -4,28 +4,35 @@ const fs = require('fs');
 const meow = require('meow');
 const prettyBytes = require('pretty-bytes');
 const gzipSize = require('gzip-size');
+const chalk = require('chalk');
 
 const cli = meow(`
 	Usage
 	  $ gzip-size <file>
 	  $ cat <file> | gzip-size
 
 	Options
-	  --level  Compression level [0-9] (Default: 9)
-	  --raw    Display value in bytes
+	  --level             Compression level [0-9] (Default: 9)
+	  --raw               Display value in bytes
+	  --include-original  Display original size
```

Comment:
The description makes it sound like it *only* displays the original size.

Reply:
Is "Include original size" would be good enough or would you want to suggest any other?

Reply:
Yeah, that's fine.

Reply:
done